test/controllers/EditSetRecordControllerSpec.scala

```diff
@@ -410,7 +414,25 @@ class EditSetRecordControllerSpec extends BaseSpec {
           )
         )
       }
-
+      "all data is valid with no record type suffix" in {
+        val oci = "COAL.2022.V2RJW"
+        val getRecordResult = getRecordForEditingWhileLoggedIn(1, oci)
+        assertPageAsExpected(
+          asDocument(getRecordResult),
+          generateExpectedEditRecordPageFromRecord(oci).copy(
+            headingPacId = s"PAC-ID: $oci",
+            optionsForCreators = Seq(
```

Comment:
We probably don't need to include 'optionsForCreators'; the important part to assert is the `headingPacId`, which you already have.

Reply:
That's true but the test expects the default `Select creator` as the selected option or it fails with _We expected 0 creator selections, but there were actually 1._

Reply:
Ah - you found a bug.

Starting on line 2639, this will fix it:

```
      optionsForCreators = {
        val recognisedCreatorIds = editSetRecord.creatorIDs.filter(creatorId => allCreators.exists(_.id == creatorId))
        val correctedCreatorIds = if (recognisedCreatorIds.nonEmpty) recognisedCreatorIds else Seq("")
        correctedCreatorIds
        .map(creatorId =>
          Seq(
            ExpectedSelectOption("", "Select creator", disabled = true),
            ExpectedSelectOption("48N", "Baden-Powell, Lady Olave St Clair (b.1889 - d.1977)"),
            ExpectedSelectOption("46F", "Fawkes, Guy (b.1570 - d.1606)", selected = true),
            ExpectedSelectOption("92W", "Joint Milk Quality Committee (1948 - 1948)"),
            ExpectedSelectOption("8R6", "Queen Anne's Bounty")
          ).map(expectedSelectedOption =>
            expectedSelectedOption.copy(selected = expectedSelectedOption.value == creatorId)
          )
        )},
```

Or adapt as you wish.

Reply:
Confirmed.